### DISCUSSION TITLE
刚入坑，启动不了游戏QAQ，求救

### DISCUSSION THREAD

**Discussion by Remiao233** [Meloong-Git/PCL#1928] [提问] [answered]
[错误报告-30-5-2023_下午4.40.25.zip](https://github.com/Hex-Dragon/PCL2/files/11598070/-30-5-2023_.4.40.25.zip)

**Comment by Remiao233** [comment]
[游戏崩溃前的输出.txt](https://github.com/Hex-Dragon/PCL2/files/11598078/default.txt)

**Reply by Remiao233** [reply]
我用的自动安装

**Comment by Remiao233** [comment]
刚试了一下不下载fabric就可以启动游戏了

**Reply by Remiao233** [reply]
但是不用fabric怎么用光影啊

**Reply by iPodtouch6557** [reply]
我建议你用edit……连发消息炸邮箱，~尤其是我在玩烫手山芋时~

**Comment by WForst-Breeze** [comment, marked as answer]
我注意到了您在尝试使用 Optifine 来在 Fabric 下加载光影，但您只安装了 Optifabric，这只是一个兼容性模组，没有 Optifine 的实质性功能，作者也在[该模组页面](https://www.curseforge.com/minecraft/mc-mods/optifabric "作者备注原文为：This project does not contain OptiFine, you must download it separately!")明确说明了这一点。因此您需要在此基础上安装 Optifine 才可以在游戏启动后安装光影。
```
	- fabricloader 0.14.21
	- java 17
	- minecraft 1.19.4
	- optifabric 1.13.24
```
与使用 Fabric 加载 Optifine 相比，我更推荐您直接在 **下载 - 1.19.4 - Optifine** 中直接安装 Optifine 而无需安装 Fabric。鉴于您没有安装其他模组，我更推荐您使用这个方法来安装 Optifine，这会避免很多游戏崩溃问题。如果您希望有更多的优化需求，可以尝试使用 Sodium+Iris 组合，不过这就麻烦了许多，因此直接安装 Optifine 是最佳选择🥳

**Reply by MoYuan-CN** [reply]
赞同，不使用模组的话单独安装 Optifine 即可驱动光影
或者也可以 [加载器]Fabric+[模组]Soudim+[模组]Iris
